src/models/trianglesSort/triangles.go

```diff
@@ -0,0 +1,109 @@
+package trianglesSort
+
+import (
+	"math"
+	"sort"
+	"errors"
+	"regexp"
+)
+
+const (
+	errorSigned    = "Numbers cant be signed, or float or 0 or string"
+	errorWrongName = "Names must contain only ABC vertices, and triangle have only 3 vertices"
+	errorInputSize = "Size of the input slice cant be 0 or 1. Can sort at least 2 triangles"
+)
+
+//Triangle is a structure, that holds Vertices names, their values and their sqrt
+type Triangle struct {
+	Vertices string  `json:"vertices"`
+	A        float64 `json:"a"`
+	B        float64 `json:"b"`
+	C        float64 `json:"c"`
+	Sqrt     float64 `json:"square,-"`
+}
+
+// TrianglesSquareSort is a function that sorts triangles slice by single triangle area. Returns sorted slice and error
+func TrianglesSquareSort(trianglesToSortSlice []Triangle) ([]Triangle, error) {
```

Comment:
named return rules here

Reply:
i'm returning the same slice as i have for my input, so i found it not necessary (if its named then then i need to reassign old to new which). So should i?